test/Microsoft.ML.OnnxTransformerTest/OnnxTransformTests.cs

```diff
@@ -607,5 +635,142 @@ public void SmokeInMemoryOnnxMapTypeTest()
                     Assert.Equal(pair.Value, transformedDataPoints[i].Output[pair.Key + 1]);
             }
         }
+
+        /// <summary>
+        /// A test to check if dynamic shape works.
+        /// </summary>
+        [OnnxFact]
+        public void TestOnnxTransformWithCustomShapes()
+        {
+            // The loaded model has input shape [-1, 3] and output shape [-1].
+            var modelFile = Path.Combine(Directory.GetCurrentDirectory(), "unknowndimensions", "test_unknowndimensions_float.onnx");
```

Comment:
>test_unknowndimensions_float [](start = 96, length = 28)

Could we use CSharp style for file names?

Reply:
It's in another repo and I feel it looks just fine.

---
In reply to: [301724967](https://github.com/dotnet/machinelearning/pull/3963#discussion_r301724967) [](ancestors = 301724967)